frontend/src/MakerApp.tsx

```diff
@@ -74,67 +73,70 @@ export default function App() {
         !runningStates.includes(value.state) && !closedStates.includes(value.state) && !openStates.includes(value.state)
     );
 
+    const labelWidth = 110;
```

Comment:
I _think_ grids would solve this problem because the column would size itself to the longest row or at least you wouldn't need to set the length at every row.

Reply:
as discussed: I could not easily figure out how to create a grid with a fixed sized column. It was always a relation between the columns (e.g. 1:3), which is not what I want I'm fairly sure it *can* be done, but I would not invest too much time into it atm.